catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The smallest acceptable size for a thumbnail is 600px at the longest edge (see
+https://github.com/WordPress/openverse/issues/675#issuecomment-1472399310)
 
 # width
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The width of the main image in pixels.
 
 # height
 
 ## Description
 
-## Object Shape
+The height of the main image in pixels.
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+If the provider does not provide the height and width of the image, it is
+possible to send a head request to the direct url to extract this dat.
 
 # filesize
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The size of the main media file in bytes. If not available in the API response,
+it can be extracted from a head request response to the media file URL.
 
 # license
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The slug of the license under which the media item is licensed. For the list of
+available license slugs, see
+[openverse-attribution package](https://github.com/WordPress/openverse/blob/3329d307f5cdce0cbe1f1c13dbe17106970abdaa/packages/python/openverse-attribution/src/openverse_attribution/license_name.py)
 
 # license_version
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The string representing the version of the license. PublicDomain has no version
+which is denoted as "N/A".
 
 # creator
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The name of the creator of the media item. Some providers use "Unknown" or
+similar for unknown creators, see
+[issue #1326](https://github.com/WordPress/openverse/issues/1326).
 
 # creator_url
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The URL of the creator's page, usually on the source site.
 
 # title
 
 ## Description
 
-## Object Shape
+The title of the media item.
 
-## Selection Criteria
+## Existing Data Inconsistencies
 
-## Normalization and Validation
+Provider scripts may include html tags in record titles, see
+[issue #1441](https://github.com/WordPress/openverse/issues/1441)
 
-## Existing Data Inconsistencies
+Some Wikimedia titles in the database still include "FILE:" prefix, and
+unnecessary file extension, which is
+[hot-fixed](https://github.com/WordPress/openverse/blob/3329d307f5cdce0cbe1f1c13dbe17106970abdaa/frontend/src/utils/decode-media-data.ts#L50)
+in the frontend. Some titles were incorrectly decoded, for which there is a
+hot-fix in the frontend
```

Comment:
Thanks for including all this! Is there an example for this last case?
```suggestion
in the frontend. Some titles were incorrectly decoded, for which there is a
hot-fix in the frontend.
```

Reply:
I searched in the Elasticsearch in prod, and couldn't find any items that still had the incorrect encoding in the title (but there are items with incorrect encodings in tags/descriptions).
I think there could be some items that still have incorrectly-encoded titles that are set as deleted from source and are only in the database, not in ES.
I wonder if we can remove the hot-fix from the frontend or if we should run the cleanup step for the title somehow (whether through the data normalization process or through the batched update).